### ISSUE TITLE
Feature request: flag to disable progress bar, for scripting

### ISSUE THREAD

**Issue by pzmarzly** [daveallie/grapple#2]
E.g. if someone would to integrate grapple into GUI application, it would be nice to have an option to switch grapple output into something like that:

```text
$ grapple FILE --progress-as-dots
......
```

with every dot representing 1 completion% (currently the topmost bar in the output), and everything else representing error text. Grapple output could then be parsed by other application that has its own system for progress bars (e.g. GTK+ progress bar).

**Comment by daveallie** [comment]
Yeah would definitely agree, I'd like to also route the progress bars through STDERR instead of STDOUT. Additionally I would like the bars to be reused (a la docker loading bars). Turning off the progress bars should be easy on its own though.
